src/modelling_utilities/extract_dynamic_images.cxx

```diff
@@ -0,0 +1,113 @@
+//
+//
+/*
+  Copyright (C) 2018, University College London
+  This file is part of STIR.
+
+  This file is free software; you can redistribute it and/or modify
+  it under the terms of the GNU Lesser General Public License as published by
+  the Free Software Foundation; either version 2.1 of the License, or
+  (at your option) any later version.
+
+  This file is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU Lesser General Public License for more details.
+
+  See STIR/LICENSE.txt for details
+*/
+
+/*!
+  \file
+  \ingroup utilities
+  \brief Split dynamic image into individual images
+  \author Richard Brown
+
+  \par Usage:
+  \code 
+  split_dynamic_images output_filename input output_parameter_file
+
+  The output filename should look something like this: dyn_im_%s,
```

Comment:
possibly `%s` works, but it's a bit weird as its argument is an int. I'd say `dyn_img_%d`. I'd also add a sentence that it's recommended to use an extension (but it might depend on the output file format used).

also call 2nd arg `output_filename_pattern` and last `output_format_parameter_file`.